packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -11,28 +10,21 @@
             {{ genericErrorMessage }}
         </alert>
         <card
-            :card-heading="title"
             is-rounded
             has-outline
             is-page-content-wrapper
             card-heading-position="center"
             data-test-id="checkout-card-component"
-            :class="$style['c-card--dimensions']">
+            :class="$style['c-checkout-card']">
```

Comment:
I'd probably make this classname `c-checkout`, if the styles/class above have been deleted. Otherwise there isn't a `c-checkout` base class anymore.